conf/content_host.yaml.template

```diff
@@ -1,33 +1,52 @@
 content_host:
-    default_rhel_version: 7
-    default_deploy_workflow: deploy-base-rhel
-    deploy_kwargs:
-        RHEL6:
-            RELEASE: 6.10
-            MEMORY: 1GiB
-            CORES: 1
-        RHEL7:
-            RELEASE: 7.9
-            MEMORY: 1GiB
-            CORES: 1
-        RHEL8:
-            RELEASE: 8.3
-            MEMORY: 1536 MiB
-            CORES: 1
-        RHEL9:
-            RELEASE: 9.0
-            COMPOSE: RHEL-9.0.0-20220118.1
-            MEMORY: 1536 MiB
-            CORES: 1
-        CENTOS7:
-            deploy_workflow: deploy-centos
-            scenario: centos
-            release: '7'
-            MEMORY: 1GiB
-            CORES: 1
-        ORACLE7:
-            deploy_workflow: deploy-oracle-linux
-            scenario: oracle
-            release: '7.9'
-            MEMORY: 1GiB
-            CORES: 1
+  default_rhel_version: 7
+  rhel6:
+    vm:
+      workflow: deploy-base-rhel
+      release: '6.10'
+      memory: 1GiB
+      cores: 1
+    container:
+      container_host: "@format {this.container_repo.registries.auto_hub.fqdn}/rhel6:latest"
```

Comment:
a little redundancy in the naming isn't it?
how about just `host` since it is a child of `container` key?

optional: wouldn't something like `image_url` describe this better?

Reply:
`container_host` is the name of the Broker argument for its Container provider. It's the equivalent of `workflow` for AnsibleTower. In the current form, these fields require no additional processing before being passed to Broker.

Reply:
We've actually decided to strip that from the image names in general since there is little value in pushing to automation hub.